contracts/pm/ERC20TicketBroker.sol

```diff
@@ -21,6 +21,24 @@ contract ERC20TicketBroker is TicketBroker {
         token = ERC20(_token);
     }
 
+    function fundAndApproveSigners(
+        uint256 _depositAmount,
+        uint256 _penaltyEscrowAmount,
+        address[] _signers
+    )
+        external
+        checkDepositPenaltyEscrowSplit(_depositAmount, _penaltyEscrowAmount)
```

Comment:
won't this always fail because it checks against `msg.value` which won't be sufficient in an ERC20 approach?

Reply:
Right - we actually don't need to check the deposit/penalty escrow split in ERC20TicketBroker since the contract doesn't need to make sure that the correct value is included in the tx. If the user did not approve a sufficient amount of tokens to be transferred by ERC20TicketBroker to cover both the specified deposit and penalty escrow amounts, the ERC20 `transferFrom` call will fail.

I removed the modifier from ERC20TicketBroker and moved it from TicketBroker into ETHTicketBroker and LivepeerETHTicketBroker in 00f5dfc since the modifier logic does not necessarily apply to all TicketBroker implementations (i.e. ERC20TicketBroker). However, this does introduce duplication of the modifier in LivepeerETHTicketBroker and ETHTicketBroker. Let me know if you have any thoughts on a better approach here.

Reply:
We could just decide to leave the modifier in `TicketBroker` even though not every broker type needs it, if we choose to prioritize DRY (don't repeat yourself) over maximizing sub-type reuse of super type code. 

I'm leaning in that direction.. to sweeten the deal we could rename the modifier to explicitly mention it's using ETH value, to reduce the odds of misuse.

wdyt?

Reply:
I don't have any strong feelings here, so I am happy to leave the modifier in TicketBroker and update its name to reflect that it deals with ETH value.